alembic/versions/003_add_agent_id.py

```
@@ -0,0 +1,31 @@
+down_revision = '249f64166921'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # Just add the agent_id column we need
+    try:
+        op.add_column('knowledge_items', sa.Column('agent_id', sa.String(length=255), nullable=False, default='legacy'))
+    except Exception:
+        # Column might already exist
+        pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)